torch/csrc/utils/python_arg_parser.h

```diff
@@ -389,9 +390,46 @@ inline std::vector<int64_t> PythonArgs::intlist(int i) {
   return intlistWithDefault(i, signature.params[i].default_intlist);
 }
 
+static bool is_symint_node(py::handle obj) {
+      auto static tp_symn = py::type::of<c10::SymbolicIntNode>();
+      return obj.get_type().equal(tp_symn);
```

Comment:
in 1:1, we discussed that isinstance should work here but not worth debugging at the moment